See Mozilla bug #675585 - crash [@ g_slice_free1] when run firefox-bin
-no-remote -process-updates. Needs further investigation. 

--- mozilla-esr31/xpcom/glue/standalone/nsXPCOMGlue.cpp.orig	2015-06-04 17:42:52.147539933 -0700
+++ mozilla-esr31/xpcom/glue/standalone/nsXPCOMGlue.cpp	2015-06-04 17:42:52.152710273 -0700
@@ -418,6 +418,8 @@
         }
     }
 
+    static void* gthreadso = dlopen("libgthread-2.0.so.0", RTLD_GLOBAL | RTLD_LAZY);
+
     GetFrozenFunctionsFunc sym =
         (GetFrozenFunctionsFunc) GetSymbol(sTop->libHandle, LEADING_UNDERSCORE "NS_GetFrozenFunctions");
 
